src/sagemaker/tensorflow/estimator.py

```diff
@@ -46,6 +46,31 @@ def _cmd_exists(cmd):
             for path in os.environ["PATH"].split(os.pathsep)
         )
 
+    @staticmethod
+    def _sync_directories(from_directory, to_directory):
+        """Sync to_directory with from_directory by copying each file in
+        to_directory with new contents. Why do this? Because TensorBoard picks
```

Comment:
Can you add a sentence to the docstring which clarifies that this function will overwrite the contents of existing files in the to_directory, but does not delete the files? This is key because it explains why we implement it in this particular way, rather than using shutil.rmtree / shutil.copytree or similar.